### DISCUSSION TITLE
badges

### DISCUSSION THREAD

**Discussion by Brainybrian316** [community/community#18461] [Discussions] [answered]
The new badge feature is amazing. The ability to showcase those badges on our profile repository would be awesome.

**Comment by HUMBLEGROUND** [comment]
i think so too!
It makes your github profile richer

More details here:

https://github.com/Schweinepriester/github-profile-achievements

**Comment by eristhea** [comment, marked as answer]
I don't really see how this is necessary?
they display by default on your profile without any need for scrolling on most devices, showing them in your profile readme and therefore displaying your badges *twice* on your profile page just seems redundant

**Reply by Brainybrian316** [reply]
Actually your right for some reason I didn’t comprehend that part lol. I for some reason thought only I could see it but if others can there no point your right thanks bro for you input
